server.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import app from './app';
+
+dotenv.config();
+
+// connect local db for local development
+const DB = process.env.DATABASE_LOCAL_URL;
+
+mongoose
+  .connect(DB, {
+    useNewUrlParser: true,
+    useCreateIndex: true,
+    useFindAndModify: false,
+    useUnifiedTopology: true
+  })
+  .then(() => {
+    console.log('DB connection successful!');
+  })
+  .catch((err) => {
+    console.log('an erro occurred:  ', err);
+  });
+
+// START THE SERVER
```

Comment:
we can put this in one function for starting the server and call it in a different file. The value of this will be evident when testing

Reply:
Do you mean creating a file with a function to start the server, then call it here?